artifacts.go

```diff
@@ -27,6 +26,9 @@ import (
 	"github.com/urfave/cli"
 )
 
+// VERSION of the mender-artifact CLI tool
+var VERSION = "0.1"
```

Comment:
Agree, but this is a bit tricky. Maybe instead of sha being a part of the version we can use branch? But for now this will be `master` which is not optimal either.